docs/tutorials/send_receive_ingress_pem.md

```diff
@@ -0,0 +1,387 @@
+              secretName: send-receive-sslacceptor-0-svc-ing-ptls
+  brokerProperties:
+    - addressConfigurations."APP.JOBS".routingTypes=ANYCAST
+    - addressConfigurations."APP.JOBS".queueConfigs."APP.JOBS".routingType=ANYCAST
+    - addressConfigurations."APP.COMMANDS".routingTypes=MULTICAST
+EOF
+```
+
+Wait for the Broker to be ready:
+
+```console
+kubectl get ActivemqArtemis  --watch
+NAME           READY   AGE
+send-receive   True    51s
+```
+
+Check that the ingress is available and has an IP address:
+
+```console
+kubectl get ingress --show-labels
+NAME                                 CLASS   HOSTS                                                                                   ADDRESS         PORTS     AGE   LABELS
+send-receive-sslacceptor-0-svc-ing   nginx   send-receive-ss-0.send-receive-hdls-svc.send-receive-project.svc.demo.artemiscloud.io   192.168.39.68   80, 443   18s   ActiveMQArtemis=send-receive,application=send-receive-app,statefulset.kubernetes.io/pod-name=send-receive-ss-0
+
+```
+
+### Exchange messages between a producer and a consumer
+
+#### Download Artemis
+
+Download the [latest
+release](https://activemq.apache.org/components/artemis/download/) of ActiveMQ
+Artemis, decompress the tarball and locate the artemis executable.
+
+#### Download the generated PEM certificate
+
+The `artemis` client will need a certificate to open a secure connection to the
+ssl encrypted endpoint we've started in the previous step. The secret got
+generated by cert-manager and is accessible under the name
+`send-receive-sslacceptor-0-svc-ing-ptls`.
+
+To execute the following commands you'll [jq](https://jqlang.github.io/jq/).
+
+```console
+$ kubectl get secrets send-receive-sslacceptor-0-svc-ing-ptls -o json | jq -r '.data."tls.crt"' | base64 -d
+-----BEGIN CERTIFICATE-----
+MIIClzCCAj2gAwIBAgIRAMM9CevyAwrXP/KTKZdUA40wCgYIKoZIzj0EAwIwLDEq
+MCgGA1UEAxMhQXJ0ZW1pc2VDbG91ZCBzZW5kLXJlY2VpdmUgaXNzdWVyMB4XDTI0
+MDUxNDEwMzI1MVoXDTI0MDgxMjEwMzI1MVowADCCASIwDQYJKoZIhvcNAQEBBQAD
+ggEPADCCAQoCggEBAPQMxPdW+9N6rLfw7Wg8d4mSWOWKiNtSWlG2eyIXmrCnwjdo
+pghihq91MjRxVCbQBsUVFc34NQ1IsOMPetWeqXjwAIXl8hvKgyKwcS644dHOKl1U
+36etHZpm1Zb0lH+0yL1HSTmP0KwKr3FbqM3QfZ83KfDrmjMU3PrmENs4HwRmD9r5
+3Qy7AFSJo9tMkMASsXbSKuJWCp44dRzD/mC9Mu+jsUUNafIeVznTBlSSxQUsvZhE
+FvLWMIgTpV8RKijuDut/YaYmBmY0C9JKgCBbKUUA4bvIIxmdeJybYqJWDoiPzvZc
+j9Z/FByaIlFFrmv+VMjEG5LZLwKbwcZYgCoRi0MCAwEAAaOBoDCBnTAOBgNVHQ8B
+Af8EBAMCBaAwDAYDVR0TAQH/BAIwADAfBgNVHSMEGDAWgBRb8dir/2ibdJQUUliM
+ft4ZT5WbpDBcBgNVHREBAf8EUjBQgk5zZW5kLXJlY2VpdmUtc3MtMC5zZW5kLXJl
+Y2VpdmUtaGRscy1zdmMuc2VuZC1yZWNlaXZlLXByb2plY3Quc3ZjLmNsdXN0ZXIu
+bG9jYWwwCgYIKoZIzj0EAwIDSAAwRQIgXNd/dVEZ+E9qFPvWg3qlwNTnPGPXO5cg
+7aXMPOKK60UCIQDMt3N/FZWJKMeRZImkXob4+9iSgusly6pPdWwDr2IV8w==
+-----END CERTIFICATE-----
+```
+
+Store the output in the `/tmp/broker.pem` file.
+
+```console
+kubectl get secrets send-receive-sslacceptor-0-svc-ing-ptls -o json | jq -r '.data."tls.crt"' | base64 -d > /tmp/broker.pem
```

Comment:
In this case, you are adding the brokers cert to your trust store which is fine but a more general approach would be to add the issuer cert to your trust store. In that way you would be able to trust any cert signed by the issuer.
In the event that you configure a second artemis cr with the same issuer, you would be able to connect to it as it would have the same 'root' of trust. Maybe that is for another tutorial.

Reply:
That's very interesting, I need to try this out

Reply:
Indeed this worked out really well. So I've updated the tutorial to use the issuer's ca to open the connection.